rpc/wrpc/server/Cargo.toml

```diff
@@ -35,8 +35,12 @@ workflow-log.workspace = true
 workflow-rpc.workspace = true
 workflow-serializer.workspace = true
 
-[target.x86_64-unknown-linux-gnu.dependencies]
-# Adding explicitely the openssl dependency here is needed for a successful build with zigbuild
+# Adding explicitely the openssl dependency here is needed for a successful build with zigbuild and musl
```

Comment:
"zigbuild" seems to be forgotten

Reply:
What do you mean by forgotten

Reply:
I mean not used anymore so doesnt belong to comment -- just a small detail anyway

Reply:
For the CI this may be true but we figure that this should be kept in the case that one builds with zigbuild anyway. And for clarification, we add the comment explaining why and when the openssl dependency is needed.